install_config/cluster_metrics.adoc

```diff
@@ -475,6 +475,78 @@ version corresponds to the very latest version available and can cause issues if
 newer version not meant to function on the version of {product-title} you are currently running.
 ====
 
+.Prometheus Variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_prometheus_state`
+|Set variable to install and update. Remove variable to uninstall.
+
+|`openshift_prometheus_namespace`
+|Project (namespace) where the components will be deployed.
+
+|`openshift_prometheus_replicas`
+|The number of replicas for prometheus deployment.
```

Comment:
@zgalor I know we have a var for this but can we scale Prometheus in this way?  Should this maybe be left undocumented or possibly moved to a var and become 'private'?